troposphere/static/js/components/settings/AdvancedSettingsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+
 import SSHConfiguration from "components/settings/advanced/SSHConfiguration";
+import ClientCredentials from "components/settings/advanced/ClientCredentials";
```

Comment:
After speaking with @lenards -- the missing link is here: https://github.com/lenards/troposphere/blob/915be6b967aa90e84b1b3de3518e484ad4bf48b7/troposphere/static/js/components/settings/advanced/ClientCredentials.jsx#L3